app/models/pairing.rb

```diff
@@ -84,11 +84,29 @@ def side_for(player)
 
   private
 
-  def aggregate_scores
-    return unless (score1_corp.present? && score1_corp > 0) || (score1_runner.present? && score1_runner > 0) ||
-      (score2_corp.present? && score2_corp > 0) || (score2_runner.present? && score2_runner > 0)
+  def prepare_scores_before_save
```

Comment:
Looks great! Consider “normalize_scores” as an alternate method name, but it’s up to you.